x-pack/plugins/observability/public/components/custom_threshold/custom_threshold_rule_expression.tsx

```diff
@@ -277,23 +273,13 @@ export default function Expressions(props: Props) {
     [ruleParams.criteria, setRuleParams]
   );
 
-  const preFillFilterQuery = useCallback(() => {
```

Comment:
@fkanout I changed how we prefill `filter` so it matches how we save it in the rule. It will also allow us to extend this filter more easily in the future. Here is what the new structure looks like:
```
searchConfiguration: {
  index: 'metrics-*',
  query: {
    query: 'host.name: host-1',
    language: 'kuery',
  },
},
```

So we don't need to prefill the filter query separately anymore.